apps/helpers.py

```diff
@@ -205,7 +215,8 @@ def update_status_time(status_object, status_ts, event_msg=None):
 
 
 def get_URI(values):
-    URI = "https://" + values["subdomain"] + "." + values["global"]["domain"]
+    subdomain = values["subdomain"] if "subdomain" in values else "deleted"
```

Comment:
I would add a line explaining how this `deleted` mark is used

Reply:
It should not be deleted but "". I will add a comment anyway